File: raw_work/LeapYear.java
import java.util.Scanner;

public class LeapYear{
	public static void main(String args[]){
	
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Enter a year : ");
		int year = sc.nextInt();
		
		if(year%4 == 0){
			if(year%100 == 0){
				if(year%400 == 0){
					System.out.println("Centuries Leap year.");
				}else{
					System.out.println("Centuries non Leap year.");
				}
			}
			System.out.println("Leap year.");
		}else{
			System.out.println("Non-Leap year.");
		} 
	}
}

File: raw_work/C programs/LeapYear.c
#include<stdio.h>

int main(void){
	int year;
	printf("Enter a year : ");
	scanf("%d", &year);
	if(year%4 == 0){
		if(year%100 == 0){
			if(year%400 == 0){
				printf("centuries leap year.");
		}else{
			printf("centuries non leap year.");
		}
			}
		printf("%d is a leap year", year);
	}else{
		printf("%d is a non leap year", year);
	}

}
